crates/routes/src/images.rs

```diff
@@ -49,6 +56,12 @@ struct PictrsParams {
   thumbnail: Option<String>,
 }
 
+#[derive(Deserialize)]
+struct PictrsPurgeParams {
+  file: Option<String>,
+  alias: Option<String>,
+}
```

Comment:
You should be  able to make this work a bit more cleanly using enum.

https://serde.rs/enum-representations.html